src/style.css

```diff
@@ -0,0 +1,139 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  /*font-family: Avenir, serif;*/
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search {
+  display: block;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+  position: relative;
+  margin: 20px 0 0 ;
+  /*box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);*/
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search__img {
+  position: absolute;
+  top: 24px;
+  left: 25px;
+}
+
+.search__area {
+  border: none;
+  padding: 0;
+  /*position: absolute;
+  top: 24px;
+  left: 61px;*/
+  margin: 23px 0 0 62px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: bold;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-family: "Avenir", Times, serif;
+}
+
+.search__area:focus {
+  outline: none;
+  font-weight: bold;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search__area::placeholder {
+  color: #3d4e61;
+  font-weight: normal;
+  text-shadow: none;
+  font-family: "Avenir", Times, serif;
+}
+
+.search__area:focus::placeholder {
+  font-weight: normal;
+  text-shadow: none;
+  color: #3d4e61;
```

Comment:
You can move value to the CSS variable (--color-main, for instance) if you use it more than one time